### DISCUSSION TITLE
Storysource can not visible

### DISCUSSION THREAD

**Discussion by e-jigsaw** [storybookjs/storybook#12379] [Help]
I use Storybook with set some parameters function like

```js
export default config({
  title: 'foo'
})
```

it will disappear codes in storysource tab.

But this modify to

```js
export default {
  title: 'foo'
}
```

is works well.

Can I use storysource with `config` like function?

Thanks

**Comment by shilman** [comment]
Nope. CSF requires statically analyzable exports, and so does source-loader.
